blog-posts/cdk/portfolio/article.md

```diff
@@ -0,0 +1,228 @@
+---
+published: false
+title: 'Easily deploy your portfolio website with AWS CDK 😎'
+cover_image:
+description: ''
+tags: serverless, tutorial, frontend, aws
+canonical_url:
+---
+
+## TL;DR
+
+In this article, I am going to explain how to deploy a portfolio website (can be plain html+css, built react etc...) to the AWS cloud. The aim is to be able to access this website using my own domain name: https://www.pchol.fr, and a HTTPS endpoint, in order to meet the current industry standards.
```

Comment:
Quelques suggestions : 
- "In this article, I am going to" -> c'est très verbeux et formel (C'est ma première impression en tout cas) : utiliser le présent plutôt ?
- Le de https://www.pchol.fr, c'est de rediriger vers ta page ? Si oui, tu peux mettre un lien (et ajouter du contenu sur ta page ? 😇 ), si non, attention car tout le monde va aller check ton site 😄

Reply:
faut que je fasse un mini portfolio sur mon site ce sera fait d'ici mercredi

Reply:
pas l'air d'avoir besoin de mettre un lien explicite en fait !